optaplanner-docs/src/main/docbook/en-US/Chapter-Planner_configuration/Chapter-Planner_configuration.xml

```diff
@@ -1400,6 +1400,15 @@ public class CourseSchedule implements Solution&lt;HardSoftScore&gt; {
 
           <para>For example in examination, a cached problem fact <literal>TopicConflict</literal> is created for every
           2 <literal>Topic</literal>'s which share at least 1 <literal>Student</literal>.</para>
+=======
```

Comment:
This line is from a merge conflict. This is the second part, up to the `>>>>>`
Please merge the 2 peices of text manually and delete the >> and == lines.